### PR TITLE
Added placeholder for carousel image tagging

### PR DESCRIPTION
**Pull request by paudyalhari**
[Card](https://rentpath.leankit.com/card/610515393)

* Added onImageClick function to handle only image click

### CODE REVIEW

**Comment by tadjohnston** [comment]
Why can't you just use `onClick` which would already work?

File: packages/react-ui-core/src/Carousel/Carousel.js [hunk 1]
```diff
@@ -70,6 +71,12 @@ export default class Carousel extends Component {
     }
   }
 
+  @autobind
+  onImageClick() {
+    this.props.onImageClick()
+    console.log('Inside carousel')
+  }
+
```
Comment (bethstevic) [inline packages/react-ui-core/src/Carousel/Carousel.js]: This won't be needed. It can be done in ag.js

Comment (bethstevic) [reply packages/react-ui-core/src/Carousel/Carousel.js]: I am about to push a PR to take care of that

**Comment by paudyalhari** [comment]
@tadjohnston actually the existing onClick is not only for image, its handling the whole card onClick which is triggering the 'image' tag while clicking on PropertyName too. but I guess I found the another solution.

**Comment by paudyalhari** [comment]
Closing this pr since Beth already created another PR for propertyName

**Pull request closed without merging**